src/arviz_plots/plots/pairs_focus_plot.py

```diff
@@ -0,0 +1,222 @@
+        * title -> :func:`~.visuals.labelled_title`
+
+    pc_kwargs : mapping
+        Passed to :class:`arviz_plots.PlotCollection`
+
+    Returns
+    -------
+    PlotCollection
+
+    Examples
+    --------
+    Default plot_pair_focus
+
+    .. plot::
+        :context: close-figs
+
+        >>> from arviz_plots import plot_pairs_focus, style
+        >>> style.use("arviz-variat")
+        >>> from arviz_base import load_arviz_data
+        >>> dt = load_arviz_data('centered_eight')
+        >>> plot_pairs_focus(
+        >>>     dt,
+        >>>     var_names=["theta", "tau"],
+        >>>     focus_var="mu",
+        >>> )
+
+    """
+    if sample_dims is None:
+        sample_dims = rcParams["data.sample_dims"]
+    if isinstance(sample_dims, str):
+        sample_dims = [sample_dims]
+    if visuals is None:
+        visuals = {}
+    if pc_kwargs is None:
+        pc_kwargs = {}
+    else:
+        pc_kwargs = pc_kwargs.copy()
+
+    if aes_by_visuals is None:
+        aes_by_visuals = {}
+    else:
+        aes_by_visuals = aes_by_visuals.copy()
+
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    if var_names is None:
+        var_names = "~" + focus_var
+
+    distribution = process_group_variables_coords(
+        dt, group=group, var_names=var_names, filter_vars=filter_vars, coords=coords
+    )
+
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+
+    if plot_collection is None:
+        pc_kwargs.setdefault(
+            "cols", ["__variable__"] + [dim for dim in distribution.dims if dim not in sample_dims]
+        )
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs = set_wrap_layout(pc_kwargs, plot_bknd, distribution)
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+        if "chain" in distribution:
+            pc_kwargs["aes"].setdefault("overlay", ["chain"])
+        pc_kwargs["figure_kwargs"].setdefault("sharey", True)
+        plot_collection = PlotCollection.wrap(
+            distribution,
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    # scatter
+    y = (
+        dt.posterior[focus_var].sel(focus_var_coords)
+        if focus_var_coords is not None
+        else dt.posterior[focus_var]
+    )
+    aes_by_visuals["scatter"] = {"overlay"}.union(aes_by_visuals.get("scatter", {}))
+    scatter_kwargs = copy(visuals.get("scatter", {}))
+    scatter_kwargs.setdefault("alpha", 0.5)
+    colors = plot_bknd.get_default_aes("color", 1, {})
+    scatter_kwargs.setdefault("color", colors[0])
+    scatter_kwargs.setdefault("width", 0)
+    _, _, scatter_ignore = filter_aes(plot_collection, aes_by_visuals, "scatter", sample_dims)
+
+    plot_collection.map(
+        scatter_x,
+        "scatter",
+        ignore_aes=scatter_ignore,
+        y=y,
+        **scatter_kwargs,
+    )
```

Comment:
It probably feels a bit pointless right now, especially in this function that currently has very little elements, but it will get more elements at some point, 2d KDEs (countour only or filled) and hexbin plots, so we might want to show some combination of elements and not some others.

Consequently, this should follow the pattern in https://arviz-plots.readthedocs.io/en/latest/contributing/new_plot.html#adding-visuals-to-the-plot.